gui.py

```diff
@@ -5,56 +5,84 @@
 import PySimpleGUI as sg
 
 from db import store_translation, retrieve_translations
-from models import Translation
 from translate import translate_text
 
+import pyperclip
+import math
+
+SEPARATOR = "→"
+
 # Set a custom theme with larger fonts
 sg.set_options(font=("Any 16"))
 
-default_lang_code = config("ITALKI_ACTIVATE_LANGUAGE_CODE", default="en")
-default_lang_name = LANGUAGES[default_lang_code]
+DEFAULT_LANG_CODE = config("ITALKI_ACTIVATE_LANGUAGE_CODE", default="en")
+DEFAULT_LANG_NAME = LANGUAGES[DEFAULT_LANG_CODE]
 
-layout = [
+LAYOUT = [
     [
         sg.Combo(
             values=list(LANGUAGES.values()),
-            default_value=default_lang_name,
+            default_value=DEFAULT_LANG_NAME,
             key="-LANGUAGE-",
             enable_events=True,
             readonly=True,
         ),
         sg.Button("Translate", bind_return_key=True),
     ],
     [sg.Input(key="-TEXT-", size=(50, 1))],
-    [sg.Multiline(key="-OUTPUT-", size=(50, 10), disabled=True, autoscroll=True)],
+    [sg.Listbox(key="-OUT LIST-", size=(50, 10), values=[])],
+    [sg.Button("Copy Selected Item", key="-COPY-")],
 ]
 
-window = sg.Window("Italki Class Assistant", layout)
+WINDOW = sg.Window("Italki Class Assistant", LAYOUT)
+
+
+def update_output(lang_code):
+    translations = retrieve_translations(lang_code)
+
+    output_values = []
+    for translation in translations:
+        output_values.append(
+            f"{translation.text} {SEPARATOR} {translation.translated_text}"
+        )
+
+    WINDOW["-OUT LIST-"].update(output_values)
```

Comment:
I would do this outside the function in main(), what about?

```
output_values = update_output(lang_code)
WINDOW["-OUT LIST-"].update(output_values)
```

Reply:
Update: I see we do this twice so this is fine then, it was just my hesitation accessing a global object.